ngboost/distns/cauchy.py

```diff
@@ -0,0 +1,14 @@
+from .t import TFixedDfFixedVar, TFixedDf, TFixedDfFixedVarLogScore, TFixedDfLogScore
```

Comment:
```suggestion
from ngboost.distns.t import TFixedDfFixedVar, TFixedDf, TFixedDfFixedVarLogScore, TFixedDfLogScore
```

this import should be absolute though, only the `__init__.py` files should have relative imports

Reply:
great catch!